services/gateway/sources/version3/mappings/networkStatistics.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+module.exports = {
+	amount: '=,string',
+	tokenID: '=,string',
+};
```

Comment:
The name and the content do not seem to match.

Reply:
renamed to `totalItem`